src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -1071,6 +1075,38 @@ internal bool OnPointerUp(PointerRoutedEventArgs args, BubblingContext ctx = def
 			return handledInManaged;
 		}
 
+		private void UpdateRaisedEventFlags(PointerRoutedEventArgs args)
```

Comment:
Please move this at the end of the _"Gestures (recognizer settings / custom bubbling / early completion)"_ region